docs/develop/typescript/data-encryption.mdx

```diff
@@ -60,6 +148,10 @@ interface PayloadConverter {
 }
 ```
 
+This custom converter should include logic for both serialization (`toPayload`) and deserialization (`fromPayload`), handling your specific data types or serialization format. The method `toPayload` returns a Payload object, which is used to manage and transport serialized data. The method `fromPayload` returns the deserialized data. This ensures that the data returned is in the same format as it was before serialization, allowing it to be used directly in the application.
+
+2. Configure the Data Converter.  To send values that are not JSON-serializable like a `BigInt` or `Date`, provide the custom Data Converter to the Client and Worker as described in the [Composite Data Converters](#composite-data-converters) section.
```

Comment:
Extra spaces here? 

Also might help to introduce a reason before sending those values. Some values cannot be expressed in JSON primitives like BigInt or Date. To send values like these, which are not JSON-serializable, ....